### PR TITLE
Created fabius user (raspberry pi 5). Changed flake.nix.

### PR DESCRIPTION
**Pull request by DerDennisOP**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -173,20 +186,25 @@
               ] else [
                 ./users/${builtins.head users}/systems/${hostname}/configuration.nix
                 ./users/${builtins.head users}/systems/${hostname}/hardware.nix
-              ]) ++ modules
-              ++ fileList "modules"
+              ]) ++ fileList "modules"
+              ++ modules
```
Comment (RichieCahill) [inline flake.nix:190]: I dont understand this change

**Review by RichieCahill — APPROVED** [review]

**Review by RichieCahill — CHANGES_REQUESTED** [review]
Move formatting into a separate PR

File: flake.nix (hunk 1 above)
Comment (DerDennisOP) [reply flake.nix:189]: https://nixos.org/manual/nix/stable/language/index.html

**Comment by DerDennisOP** [comment]
image

NIX FLAKE CHECK HOOK REMOVED!

**Review by ahuston-0 — COMMENTED** [review]
flake.nix stuff isn't super clear to me bc I'm on mobile and I see Richie requested changes... Added one comment but otherwise I'll defer to their approval.

File: users/dennis/systems/configuration.nix [hunk 2]
```diff
@@ -266,6 +266,4 @@
     hitori # sudoku game
     atomix # puzzle game
   ]);
-
-  system.stateVersion = "22.11";
```
[the remark this answers is not in the document]
Comment (DerDennisOP) [reply users/dennis/systems/configuration.nix:270]: thats was an error on my side, It got actually moved to systems/DennisLaptop and DennisMain

File: flake.nix [hunk 3]
```diff
@@ -173,21 +185,27 @@
...
+                boot.initrd.network.ssh.authorizedKeys = lib.mkIf server config.users.users.${user}.openssh.authorizedKeys.keys;
+                sops = lib.mkIf sops {
+                  secrets."${user}/user-password" = {
+                    sopsFile = ./users/${user}/secrets.yaml;
+                    neededForUsers = true;
                   };
-                })
-                users;
+                };
+              }) users;
```
Comment (RichieCahill) [inline flake.nix:208]: formating

File: flake.nix [hunk 4]
```diff
@@ -160,11 +169,14 @@
...
+          constructSystem = { hostname, users, home ? true, iso ? [ ], modules ? [ ], server ? true, sops ? true, system ? "x86_64-linux" }:
             lib.nixosSystem {
               inherit system;
-
-              modules = [ nixos-modules.nixosModule sops-nix.nixosModules.sops { config.networking.hostName = "${hostname}"; } ] ++ (if server then [
+              modules = [
+                nixos-modules.nixosModule
+                sops-nix.nixosModules.sops
+                { config.networking.hostName = "${hostname}"; }
+              ] ++ (if server then [
```
Comment (RichieCahill) [inline flake.nix:179]: formating

File: flake.nix (hunk 1 above)
Comment (DerDennisOP) [reply flake.nix:189]: those change would no formatter in the world do, because it would break code

**Review by RichieCahill — APPROVED** [review]

**Pull request merged**